crates/ide-completion/src/completions/postfix.rs

```diff
@@ -84,6 +89,17 @@ pub(crate) fn complete_postfix(
                 )
                 .add_to(acc, ctx.db);
 
+                postfix_snippet(
+                    "lete",
+                    "let Ok else {}",
+                    &if in_loop {
+                        format!("let Ok($1) = {receiver_text} else {{\n    ${{2|continue,break,return|}};\n}};\n$0")
+                    } else {
+                        format!("let Ok($1) = {receiver_text} else {{\n    ${{2:return}};\n}};\n$0")
+                    },
```

Comment:
```suggestion
                    &format!("let Ok($1) = {receiver_text} else {{\n    $0\n}};"),
```
Let's not prefill these, there are more options people could strife for. Same for the option one below

Reply:
I'm a bit sad to lose the convenience, but I see how these could be annoying when you want to do more complex things.